pkg/addons/adminconsole/customize.go

```diff
@@ -170,6 +178,28 @@ func (a *AdminConsoleCustomization) apply(ctx context.Context) error {
 	return nil
 }
 
+// clusterConfig makes sure a config map named "embedded-cluster-config" exists in the cluster.
+// XXX this will eventually be customizable by the user through the app release.
+func (a *AdminConsoleCustomization) clusterConfig(ctx context.Context) error {
+	kubeclient, err := a.kubeClient()
+	if err != nil {
+		return fmt.Errorf("unable to create kubernetes client: %w", err)
+	}
+	nsn := client.ObjectKey{Namespace: "kube-system", Name: "embedded-cluster-config"}
```

Comment:
https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/types.go#L35

should we use that const?

Reply:
importing kubernetes/kubernetes is not to be done lightly IIRC